### PR TITLE
Added identity hack panel discussion blog

### PR DESCRIPTION
**Pull request by pekkakaipio-op**
Added identity hack panel discussion blog with custom youtube wrapper

### CODE REVIEW

File: content/en/blog/identity-hack-panel-discussion/index.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+linkTitle: "Digital Identity Hack Part 2 – Panel Discussion on Blockchain Projects"
+description: "In this blog, we discuss about the Digital Identity Hack event's panel discussion, which focused on exploring development of blockchain projects in various companies."
+author: Pekka Kaipio
+resources:
+  - src: "**.{png,jpg}"
+    title: "Image #:counter"
+---
+The emergence of **self-sovereign identity** (SSI) offers a revolutionary
+opportunity for innovation and growth in the financial world, with the potential
+to transform digital financial services, streamline transactions, and enable
```
Comment (lainio) [inline content/en/blog/identity-hack-panel-discussion/index.md:13]: with the potential... -> potentially transforming digital financial services, streamlining transactions, and enabling seamless interaction.

File: content/en/blog/identity-hack-panel-discussion/index.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+seamless interactions. To harness this potential, OP Lab and the EU-funded
+Powered by Blockchain project recently held a seminar day and online hackathon
+to investigate the disruptive capabilities of SSI technologies and institutional
+decentralized finance, paving the way for **a more secure, inclusive, and
+efficient global financial ecosystem.** 
+
+During the seminar event, attendees participated in a panel discussion focused
```
Comment (lainio) [inline content/en/blog/identity-hack-panel-discussion/index.md:20]: -> During the seminar, attendees participated in a panel discussion on blockchain projects and a presentation exploring self-sovereign identity. The following section will delve into the key insights and highlights from the panel discussion

File: content/en/blog/identity-hack-panel-discussion/index.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+Many blockchain projects are built upon open-source technologies like
+Hyperledger Fabric. Panelists highlighted the importance of open-source
+contributions in **advancing blockchain technology and fostering a spirit of
+collaboration** within the community. While not all panelists' companies have
+made code contributions, they have shared knowledge and participated in meetups
+and user groups. 
+
+### Effective Talent Sourcing 
+
+Sourcing skilled blockchain talent has proven to be difficult. Panelists
```
Comment (lainio) [inline content/en/blog/identity-hack-panel-discussion/index.md:81]: difficult -> complicated

File: content/en/blog/identity-hack-panel-discussion/index.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+as an infrastructure solution for industries like finance and the emerging
+metaverse was also noted. However, the panelists acknowledged that regulatory
+challenges and the rise of artificial intelligence (AI) could impact the
+technology's trajectory. 
+
+### One Wish for Blockchain 
+
+When asked about their one wish for blockchain, panelists expressed a desire for
+**more streamlined regulations, increased adoption, and a broader talent pool**.
+They hope that these changes will lead to a more decentralized, fair, and
```
Comment (lainio) [inline content/en/blog/identity-hack-panel-discussion/index.md:99]: that these -> these

File: content/en/blog/identity-hack-panel-discussion/index.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+trusted way for people to exchange value and information. 
+
+In conclusion, the panelists emphasized that despite its challenges,
+**blockchain has the potential to revolutionize various industries**, including
+finance, real estate, logistics, and digital identity management. By sharing
+their experiences and lessons learned, the panelists provided valuable insights
+to help others navigate the complex world of blockchain technology. 
+
+
+## Project and event information
```
Comment (lainio) [inline content/en/blog/identity-hack-panel-discussion/index.md:109]: Heading style, uppercase start letters?

File: content/en/blog/identity-hack-panel-discussion/index.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+**blockchain has the potential to revolutionize various industries**, including
+finance, real estate, logistics, and digital identity management. By sharing
+their experiences and lessons learned, the panelists provided valuable insights
+to help others navigate the complex world of blockchain technology. 
+
+
+## Project and event information 
+
+**OP Lab** arranged the seminar day in Oulu, Finland, and the two-week online
+hackathon during February in collaboration with Powered by Blockchain project.
```
Comment (lainio) [inline content/en/blog/identity-hack-panel-discussion/index.md:112]: with -> with the

File: content/en/blog/identity-hack-panel-discussion/index.md [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+**Powered by Blockchain** project is a collaboration project between University
```
Comment (lainio) [inline content/en/blog/identity-hack-panel-discussion/index.md:113]: the University, all universities should start with `the`?

**Review by lainio — APPROVED** [review]
Good stuff, with some minor language stuff as an option...

File: content/en/blog/identity-hack-panel-discussion/index.md [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+efficient global financial ecosystem.** 
+
+During the seminar event, attendees participated in a panel discussion focused
+on blockchain projects and a presentation exploring self-sovereign identity. The
+following section will delve into the key insights and highlights that emerged
+from the panel discussion. 
+
+## Panel Discussion on Blockchain Projects 
+
+{{< youtube-wrapper nygxmT3dN-g 939px 649px >}}
```
Comment (lauravuo) [inline content/en/blog/identity-hack-panel-discussion/index.md:27]: Is this new wrapper necessary? Can't the videos be embedded similarly as with other blogs (e.g. https://findy-network.github.io/blog/2022/09/27/the-hyperledger-global-forum-experience/, https://findy-network.github.io/blog/2023/05/08/deploying-with-cdk-pipeline/) ? For some reason I am unable to preview the post and start the hugo process, maybe it's due to this addition.

Comment (pekkakaipio-op) [reply content/en/blog/identity-hack-panel-discussion/index.md]: Ok removed, maybe it doesn't work with every version and might be problematic later on. The shortcode was adjusting the video width to align with the body text and images.

Comment (lauravuo) [reply content/en/blog/identity-hack-panel-discussion/index.md]: Great, thanks. It seems that my issue was another problem, but still less code to maintain in the future 😄

**Pull request merged**